component/stream/read_write.go

```diff
@@ -203,25 +202,41 @@ func (rw *ReadWriteCache) RenameFile(options internal.RenameFileOptions) error {
 	err := rw.NextComponent().RenameFile(options)
 	if err != nil {
 		log.Err("Stream::RenameFile : error renaming file %s [%s]", options.Src, err.Error())
+	}
+	return err
+}
+
+func (rw *ReadWriteCache) FlushFile(options internal.FlushFileOptions) error {
+	// log.Trace("Stream::FlushFile : name=%s, handle=%d", options.Handle.Path, options.Handle.ID)
```

Comment:
FlushFile shall work only on dirty file handles, for clean handles we can just return back success.

Reply:
storage component will just not do anything since all blocks are clean. However, I guess it would save time for the extra operations